SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -279,6 +290,20 @@ void USpatialReceiver::OnAddComponent(const Worker_AddComponentOp& Op)
 			NetDriver->VirtualWorkerTranslator->ApplyVirtualWorkerManagerData(ComponentObject);
 		}
 		return;
+	case SpatialConstants::GDK_DEBUG_COMPONENT_ID:
+		if (NetDriver->DebugCtx)
```

Comment:
another nit here and in a few other places in the pr - code guidelines say to check against nullptr